sonar-helm-for-iac/build.gradle.kts

```
@@ -103,7 +130,7 @@ if (!isCi) {
         outputs.files(fileTree("build/executable").include("sonar-helm-for-iac-*"))
         outputs.cacheIf { true }
 
-        commandLine("docker", "run", "--mount", "type=bind,source=${project.projectDir},target=/home/sonarsource/sonar-helm-for-iac", "sonar-iac-helm-builder")
+        commandLine("docker", "run", "--rm", "--mount", "type=bind,source=${project.projectDir},target=/home/sonarsource/sonar-helm-for-iac", "sonar-iac-helm-builder")
```

```suggestion
        commandLine("docker", "run", "--rm", "--platform", "linux/amd64", "--mount", "type=bind,source=${project.projectDir},target=/home/sonarsource/sonar-helm-for-iac", "sonar-iac-helm-builder")
```

Thanks to that we will skip one warning